app/views/hosts/_selected_hosts.html.erb

```diff
@@ -2,43 +2,45 @@
   <%= alert(:class => 'alert-info hide', :id => 'multiple-modal-alert', :close => true, :header => '',
             :text => multiple_filter(hosts)) %>
   <% unless multiple_with_filter? %>
-    <table class="<%= table_css_classes %>">
-      <thead>
-      <tr>
-        <th>
-          <%= _("Name") %>
-        </th>
-        <th>
-          <%= _("Host group") %>
-        </th>
-        <th>
-          <%= _("Environment") %>
-        </th>
-        <%= raw("<th>" + _("Location") + "</th>") if SETTINGS[:locations_enabled]%>
-        <%= raw("<th>" + _("Organization") + "</th>") if SETTINGS[:organizations_enabled]%>
-      </tr>
-      </thead>
-      <tbody>
-      <% associations = [:hostgroup, :environment] %>
-      <% associations << :organization if SETTINGS[:organizations_enabled]  %>
-      <% associations << :location     if SETTINGS[:locations_enabled]  %>
-      <% hosts.includes(*associations).each do |host| %>
-        <tr>
-          <td><%=h host %>
-          </td>
-          <td><%=h host.try(:hostgroup) %></td>
-          <td><%=h host.try(:environment) %></td>
-          <% if SETTINGS[:locations_enabled]%>
-            <td><%=h host.try(:location) %></td>
+    <div class="col-md-12">
+      <table class="<%= table_css_classes %>">
+        <thead>
+          <tr>
+            <th>
+              <%= _("Name") %>
+            </th>
+            <th>
+              <%= _("Host group") %>
+            </th>
+            <th>
+              <%= _("Environment") %>
+            </th>
+            <%= raw("<th>" + _("Location") + "</th>") if SETTINGS[:locations_enabled]%>
+            <%= raw("<th>" + _("Organization") + "</th>") if SETTINGS[:organizations_enabled]%>
+          </tr>
+        </thead>
+        <tbody>
+          <% associations = [:hostgroup, :environment] %>
+          <% associations << :organization if SETTINGS[:organizations_enabled]  %>
+          <% associations << :location     if SETTINGS[:locations_enabled]  %>
+          <% hosts.includes(*associations).each do |host| %>
+          <tr>
```

Comment:
indentation is a bit off, I'll fix that on merge

Reply:
sorry for that :)